aten/src/ATen/native/native_functions.yaml

```diff
@@ -2082,10 +2082,6 @@
   variants: function, method
   device_guard: False
 
-- func: is_complex(Tensor self) -> bool
-  variants: function, method
-  device_guard: False
-
```

Comment:
The error logs are suggesting that either the type hint for `is_complex` is missing, or the actual `Tensor.is_complex` method is missing.

I suspect the problem is the former. @jbschlosser could you check if `is_complex` is in `torch/_C/__init__.pyi` (this is an autogenerated file) on your local branch? If not we have to manually add that type hint somewhere
```
Feb 10 19:33:01 ======================================================================
Feb 10 19:33:01 FAIL [69.756s]: test_run_mypy (__main__.TestTypeHints) [mypy.ini]
Feb 10 19:33:01 ----------------------------------------------------------------------
Feb 10 19:33:01 Traceback (most recent call last):
Feb 10 19:33:01   File "test_type_hints.py", line 171, in test_run_mypy
Feb 10 19:33:01     self.fail(f"mypy failed: {stdout} {stderr}")
Feb 10 19:33:01 AssertionError: mypy failed: torch/testing/__init__.py:108: error: "Tensor" has no attribute "is_complex"  [attr-defined]
Feb 10 19:33:01 torch/testing/_internal/common_nn.py:5106: error: "Tensor" has no attribute "is_complex"  [attr-defined]
Feb 10 19:33:01 torch/testing/_internal/common_methods_invocations.py:691: error: "Tensor" has no attribute "is_complex"  [attr-defined]
Feb 10 19:33:01 torch/testing/_internal/common_methods_invocations.py:3308: error: "Tensor" has no attribute "is_complex"  [attr-defined]
Feb 10 19:33:01 Found 4 errors in 3 files (checked 1229 source files)
```

Reply:
Nice, thanks for catching this! I updated `tools/pyi/gen_pyi.py` to add the entry for `is_complex` and the type hint tests pass now.